### ISSUE TITLE
A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_Keyboard:There is no keyboard key to increase the size of the left panel of azure

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9257]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.1.1;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.1.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Azure, navigate to + button & hit enter.
5. Navigate to Add account button in right side window & hit enter.
6. Navigate to each item in the list.

**Actual:**
On navigating to the items the names of all the items are dotted at the end the full name is not visible since the size of the panel is small. With the help of the mouse the panel increases in size.

**Expected:**
The functionality to increase and decrease the panel size should be available through the mouse.

**User Impact:**
Keyboard users will not know the full name associated with the items since its hidden due to the size of the panel.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EaFOVxMwR_lKrT6ScsGupAsB4ekQVCxhBFnWlPAOD4CiTg?e=L0DZCB

**Comment by alanrenmsft** [comment]
this is a vscode issue, their solution to this is using command to increase the view size, but the command is not working, i opened an issue https://github.com/microsoft/vscode/issues/92054

**Comment by Roopesh-T** [comment]
Issue not repro in 
![image](https://user-images.githubusercontent.com/86601127/217487321-deed1d6b-22e4-43c2-9ba7-b31eb62c8409.png)
so closing the bug.
![9257_Closed](https://user-images.githubusercontent.com/86601127/217487405-0078c224-d4a4-4a8b-91fe-83683602193d.gif)

**Git Hub**: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
